guides/v2.1/extension-dev-guide/build/module-file-structure.md

```diff
@@ -34,6 +34,7 @@ Following are some common module directories:
 Additional folders can be added for configuration and other ancillary functions for items like [plugin-ins]({{ page.baseurl }}/extension-dev-guide/plugins.html), localization, and {% glossarytooltip 73ab5daa-5857-4039-97df-11269b626134 %}layout{% endglossarytooltip %} files.
 
 * `Api`: contains any PHP classes exposed to the {% glossarytooltip 786086f2-622b-4007-97fe-2c19e5283035 %}API{% endglossarytooltip %}.
+* `Console`: contains files for creating CLI commands.
```

Comment:
I'd suggest to follow the same approach as with plugins. Something like `contains CLI commands` with a reference to https://devdocs.magento.com/guides/v2.2/extension-dev-guide/cli-cmds/cli-add.html

Reply:
Done. Thanks!